rust/protocol/src/sealed_sender.rs

```diff
@@ -704,6 +803,58 @@ pub async fn sealed_sender_encrypt<R: Rng + CryptoRng>(
     sealed_sender_encrypt_from_usmc(destination, &usmc, identity_store, ctx, rng).await
 }
 
+/// This method implements a single-key single-recipient [KEM], described in [this Signal blog post]
+/// on the sealed sender mechanism.
+///
+/// [KEM]: https://en.wikipedia.org/wiki/Key_encapsulation
+/// [this Signal blog post]: https://signal.org/blog/sealed-sender/
+///
+/// ### Contrast with Multi-Recipient KEM
```

Comment:
Extreme nitpick: the rustdoc book seems to suggest starting at `#` for section headers, even if it's pretty strong.

Reply:
Done! I believe it is possible to edit the CSS used in a generated docsite if we find it too aggressive (it seems mostly fine to look at by default though), although I haven't done so myself yet.